optimade/server/middleware.py

```diff
@@ -297,6 +297,14 @@ class AddWarnings(BaseHTTPMiddleware):
     the response's body content is actually streamable, by breaking it down into
     chunks of the original response's chunk size.
 
+    !!! warning "Important"
+        It is **recommended** to add this middleware as the _last one_ to your application.
+
+        This is to ensure it is invoked _first_, updating `warnings.showwarning()` and
```

Comment:
Is _first_ correct here? It's not clear to me from the Starlette docs which way this goes... Is there a distinction between the discussion in those docs when initialising middleware as a list on app creation, versus the one-at-a-time approach we take? (i.e. are we putting it on the top of a stack?)

Reply:
Just played with this, whilst you're absolutely correct I think we should clarify it slightly wrt to the Starlette docs in the docstring for the `OPTIMADE_MIDDLEWARE` variable.

Reply:
Going to commit my above suggestion and tie this all together